### PR TITLE
Cookie banner docs

### PR DESCRIPTION
**Pull request by ahervy**
## Summary

This pull request adds the documentation for the cookie banner.

Story details: https://app.clubhouse.io/opendatasoft/story/27478

## Changes 

- In "Displaying legals", the ["Cookies and trackers information"](https://github.com/opendatasoft/ods-documentation/compare/feature/ch27478/cookie-banner-docs?expand=1#diff-90bb39f592643282ee065adee47d396334626de1d929dc17bea5d4027d11c4dfL4) has been replaced with a ["Cookie banner"](https://github.com/opendatasoft/ods-documentation/compare/feature/ch27478/cookie-banner-docs?expand=1#diff-90bb39f592643282ee065adee47d396334626de1d929dc17bea5d4027d11c4dfR7) section.
- The Cookie banner section includes [general info](https://github.com/opendatasoft/ods-documentation/compare/feature/ch27478/cookie-banner-docs?expand=1#diff-90bb39f592643282ee065adee47d396334626de1d929dc17bea5d4027d11c4dfR10-R12) about the cookie banner, a section about [legal compliance](https://github.com/opendatasoft/ods-documentation/compare/feature/ch27478/cookie-banner-docs?expand=1#diff-90bb39f592643282ee065adee47d396334626de1d929dc17bea5d4027d11c4dfR10-R12) to mention when a cookie banner is needed and visible for visitors after activation, and a procedure to [enable the cookie banner](https://github.com/opendatasoft/ods-documentation/compare/feature/ch27478/cookie-banner-docs?expand=1#diff-90bb39f592643282ee065adee47d396334626de1d929dc17bea5d4027d11c4dfR26-R41).
- Added info about the `##manage-cookies##` placeholder in ["Customizing portal themes"](https://github.com/opendatasoft/ods-documentation/compare/feature/ch27478/cookie-banner-docs?expand=1#diff-e74fcbf5e4dd5003ea5d2c3cc5535fd91059688d0b246ea1eb4a680e90d926e6R217-R218).
- Added a section about Matomo in ["Third-party analytics"](https://github.com/opendatasoft/ods-documentation/compare/feature/ch27478/cookie-banner-docs?expand=1#diff-c6e440d6fd65214262f0083fffdade1a620fde20b4a04f2b43186c669a247b11R66-R76) with activation steps and required information to be sent to the support team.
- Added the Require user consent option for [XiTi](https://github.com/opendatasoft/ods-documentation/commit/cdab8fc0716a1be03cbbe6ff03d3da8116b62de3#diff-c6e440d6fd65214262f0083fffdade1a620fde20b4a04f2b43186c669a247b11R39), [SmartTag](https://github.com/opendatasoft/ods-documentation/commit/cdab8fc0716a1be03cbbe6ff03d3da8116b62de3#diff-c6e440d6fd65214262f0083fffdade1a620fde20b4a04f2b43186c669a247b11R54), and [Matomo](https://github.com/opendatasoft/ods-documentation/commit/cdab8fc0716a1be03cbbe6ff03d3da8116b62de3#diff-c6e440d6fd65214262f0083fffdade1a620fde20b4a04f2b43186c669a247b11R67) in "Third-party analytics".
- Cleaned the "Third-party analytics" section–removed a [redundant note](https://github.com/opendatasoft/ods-documentation/compare/feature/ch27478/cookie-banner-docs?expand=1#diff-c6e440d6fd65214262f0083fffdade1a620fde20b4a04f2b43186c669a247b11L14-L17) from the intro, removed screenshots, and fixed wording issues (textbox > text box).

### CODE REVIEW

File: source/configuring_domain/03_managing_tracking/third_party_analytics.rst [hunk 1]
```diff
@@ -4,56 +4,45 @@ Third-party analytics
 The monitoring tools provided by Opendatasoft focus on API calls and usages.
 They are useful to gather precise information on how data is consumed (see :doc:`Analyzing domain usage </managing_domain/03_analyzing_domain_usage/index>`). But it can be limiting when trying to measure more broadly the audience of a portal: referral sources, visitors' geographical origin, time spent on the portal, etc.
 
-Third-party services, specialized in measuring audience, can give additional analytics such as those mentioned above. Opendatasoft supports integration with Google Analytics, and XiTi and SmartTag by AT Internet.
+Third-party services, specialized in measuring audience, can give additional analytics such as those mentioned above. Opendatasoft supports integration with Google Analytics, XiTi and SmartTag by AT Internet, and Matomo.
```
Comment (remilejeune) [inline source/configuring_domain/03_managing_tracking/third_party_analytics.rst:7]: "Google Analytics, XiTi and SmartTag by AT Internet, and Matomo" => 
"Google Analytics, XiTi, SmartTag by AT Internet, and Matomo"

File: source/configuring_domain/03_managing_tracking/third_party_analytics.rst [hunk 2]
```diff
@@ -4,56 +4,45 @@ Third-party analytics
...
 
-.. image:: images/configuration_tracking-google-analytics.png
-
 Google Analytics is an analytics solution edited by Google.
 
 Currently, only "Universal Analytics" properties are supported.
 
 To activate and configure Google Analytics:
 
 1. In the Configuration > Tracking section of the back office, go to the Google Analytics area.
```
Comment (remilejeune) [inline source/configuring_domain/03_managing_tracking/third_party_analytics.rst:23]: Not related to that particular PR, but in the legal section, "Configuration > Tracking" is displayed in bold but not in that section.

**Review by remilejeune — CHANGES_REQUESTED** [review]

File: source/configuring_domain/03_managing_tracking/third_party_analytics.rst (hunk 1 above)
Comment (ahervy) [reply source/configuring_domain/03_managing_tracking/third_party_analytics.rst]: [Fixed](https://github.com/opendatasoft/ods-documentation/pull/286/files#diff-c6e440d6fd65214262f0083fffdade1a620fde20b4a04f2b43186c669a247b11R7)

File: source/configuring_domain/03_managing_tracking/third_party_analytics.rst (hunk 2 above)
Comment (ahervy) [reply source/configuring_domain/03_managing_tracking/third_party_analytics.rst]: [Fixed](https://github.com/opendatasoft/ods-documentation/pull/286/commits/f643f431a598413c8536af3dd5d083e689f7b7b6#diff-c6e440d6fd65214262f0083fffdade1a620fde20b4a04f2b43186c669a247b11)
Thanks for the review!

**Review by remilejeune — APPROVED** [review]

**Pull request merged**